### PR TITLE
Added migration runner that allows for migrations to use server code

### PR DESCRIPTION
**Pull request by TheHollidayInn**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes put_issue_url_here

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

File: migrations/migration-runner.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  setupNconf();
+  // We require src/server and npt src/index because
+  // 1. nconf is already setup
+  // 2. we don't need clustering
+  require('../website/server/server'); // eslint-disable-line global-require
+}
+setUpServer();
+
+.// Replace this with your migration
+var regiserComedUsers = require('./register-users-comed');
```
Comment (SabreCat) [inline migrations/migration-runner.js]: Maybe replace this with one of our existing migrations not specific to a client, like `new_stuff`?

Comment (TheHollidayInn) [reply migrations/migration-runner.js]: The existing migrations aren't formatted correctly.

Comment (SabreCat) [reply migrations/migration-runner.js]: Then might it be feasible to update a simple existing migration to use the correct format in this PR? I feel strange referencing customer-specific code here.

Comment (TheHollidayInn) [reply migrations/migration-runner.js]: Oh! Sure. No problem.

**Comment by TheHollidayInn** [comment]
Okay, I changed the example. However, the new stuff migration isn't taking advantage of the runner, but as shown, all you have to do is export.

**Comment by SabreCat** [comment]
Failing test is just the Amazon Payments timeout issue we've been running into lately.

**Pull request merged**